src/rules.c

```diff
@@ -47,7 +57,6 @@ void SchemaRule_Free(SchemaRule *rule) {
   SchemaPrefixes_RemoveSpec(rule->spec);
   SchemaRules_RemoveSpecRules(rule->spec);
 
-  rm_free((void*) rule->type);
```

Comment:
Why not free it?

Reply:
Ohh I see it was changed to enum, I still will like to save the string format for rdbsave or we will need to have toString for the enum

Reply:
`rule->type` is an enum now.